stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -53,6 +53,7 @@ spec:
           env:
             - name: JAVA_OPTS
               value: "{{ default "" .Values.Master.JavaOpts}}"
+            {{- if .Values.Master.UseSecurity }}
```

Comment:
we'd probably want to wrap the secret as well such that `jenkins-admin-password` and `jenkins-admin-username` can be omitted (or the secret omitted entirely).